test/dotsnyk.test.ts

```diff
@@ -0,0 +1,44 @@
+import { parseDotSnyk, parseUnmanagedExcludes } from '../lib/utils/dotsnyk';
+import { join } from 'path';
+
+describe('parseDotSnyk', () => {
```

Comment:
Would it add any value to have a test where we read from an actual .snyk file to approximate how it's used in real life?